app/assets/stylesheets/runtime.scss

```diff
@@ -0,0 +1,96 @@
+// The classes in this file are all related to a Runtime Filter feature detailed
+// in RFC 27. A short summary of how they work can be summarized by:
+//
+// They all are invisible by default. Only after the `initializeRuntime()` code
+// adds the context to the data-runtime attribute to the body element is when
+// elements with these classes become visible.
+//
+// More thorough context can be found in: https://github.com/forem/rfcs/pull/27
+
+// Elements with all of these classes are not visible by default
+.ForemWebView,
+.ForemWebView-iOS,
+.ForemWebView-Android,
+.PWA,
+.PWA-iOS,
+.PWA-Android,
+.PWA-Windows,
+.PWA-Linux,
+.PWA-macOS,
+.Browser,
+.Browser-iOS,
+.Browser-Android,
+.Browser-macOS,
+.Browser-Windows,
+.Browser-Linux {
+  display: none;
+}
+
+// Elements with class 'ForemWebView' should be visible on any attribute that
+// matches that Medium regardless of OS
+body[data-runtime='ForemWebView-iOS'] .ForemWebView,
+body[data-runtime='ForemWebView-Android'] .ForemWebView {
+  display: block;
```

Comment:
What happens if one of those elements should be `display: inline` or `display: flex` or anything else? In other words: to hide an element it's obvious to use `display: none` but to show the element we may need to use different values. 

I'm also not sure if that will ever be a problem, but I'm just thinking out loud.. 

Maybe we could use a variable like

```
body[data-runtime='ForemWebView-iOS'] .ForemWebView,
... {
  display: var(--runtime-display, block);
}
```

^ this will apply `block` value unless you specify otherwise:

```
.element { // <div class="element ForemWebView" />
  --runtime-display: flex;
}
```

Reply:
+1 to this suggestion - sounds good!

Reply:
This is pretty cool! I changed it to use this, thanks @ludwiczakpawel